Crayon/Decorations.cs

```diff
@@ -3,6 +3,7 @@ namespace Crayon
     internal struct Decorations
```

Comment:
Why not use an enum here btw?

Reply:
If (I recall correctly)  then you have to explicitly cast it to an int to get the underlying value in the string conversion so it didn't make much sense to use an enum altogether.